libbpf-tools/filelife.bpf.c

```diff
@@ -37,20 +40,34 @@ probe_create(struct inode *dir, struct dentry *dentry)
 }
 
 SEC("kprobe/vfs_create")
-int BPF_KPROBE(vfs_create, struct inode *dir, struct dentry *dentry)
+int BPF_KPROBE(vfs_create, struct user_namespace *mnt_userns,
```

Comment:
This breaks the tool on old kernel. We should fix it in a CO-RE way and the fix should go into `core_fixes.h`.

Reply:
Thanks, i'll try.